packages/core/sandbox/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="icon"
+      href="data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='36' viewBox='0 0 36 36' width='36'%3E%3Cg fill='none' fill-rule='evenodd' transform='translate(0 4.544118)'%3E%3Cpath d='m24.7018129.03888403 11.2778281 6.67879663-.0181821 13.56348904-11.259646 6.6606051-6.6844666-3.9709902 11.6514714-6.541272v-5.8973849l-4.7471673-2.60470586-6.8895427-3.93711399' fill='%230095d3'/%3E%3Cpath d='m11.3313965.03888403-11.277828 6.67879663.01818205 13.56348904 11.25964595 6.6606051 6.6852924-3.9717138-10.66220196-6.5405484v-5.8973849l10.67797726-6.54221107' fill='%23f38b00'/%3E%3Cpath d='m18.017374 22.9708988-6.5183252-3.998915 6.5222007-3.8447451 6.9298332 3.951391' fill='%23004b70'/%3E%3Cpath d='m18.0314053 3.98921729-6.5046536 3.98442963 6.5172421 3.88418548 6.8619013-3.93951296' fill='%2398441e'/%3E%3C/g%3E%3C/svg%3E"
+      type="image/svg+xml"
+    />
+    <link href="./index.css" rel="stylesheet" />
+  </head>
+  <body cds-theme="dark" cds-text="body">
+    <section cds-layout="vertical gap:lg p:xl container:sm container:center">
+      <h1 cds-text="heading">Clarity Core Sandbox</h1>
+
+      <p cds-text="body">
+        This is a sandbox enviroment that allows you to test your component locally. All imports will automatically be
+        mapped to the local <code cds-text="code">dist</code> output of the build.
+      </p>
+
+      <cds-alert status="warning">Do not commit sandbox changes to source control.</cds-alert>
+
+      <cds-button status="primary">Hello There</cds-button>
+
+      <cds-alert-group status="danger" hidden>
+        <cds-alert closable>
+          General Kenobi, you are a bold one...
+        </cds-alert>
+      </cds-alert-group>
+    </section>
+
+    <script type="module" src="./index.js"></script>
+  </body>
+</html>
```

Comment:
The new sandbox app can be started by running `yarn run start:sandbox` it will auto reload anytime the build is updated via `yarn run start` or `yarn run build:watch`. This enables a lightweight dev env outside of storybook.